flash/core/data/transforms.py

```diff
@@ -73,6 +73,14 @@ class KorniaParallelTransforms(nn.Sequential):
 
     def __init__(self, *args):
         super().__init__(*[convert_to_modules(arg) for arg in args])
+        self._reuse_params: bool = False
+
+    @property
+    def reuse_params(self) -> bool:
+        return self._reuse_params
+
+    def set_reuse_params(self, val: bool):
```

Comment:
Why not use `@reuse_params.setter` here?

Reply:
if we don't have this e.g in segmentation I couldn't find a way to keep parameters to be applied to both images and masks. ok, for setter

Reply:
for the failing tests no idea - cannot reproduce in local

Reply:
Looks like you need to update the segmentation model on AWS. But the usage here applies the same transform to image and mask no? https://github.com/PyTorchLightning/lightning-flash/blob/c11498d0c12357f448dedea0fa61262714f4fa11/flash/image/segmentation/transforms.py#L55

Reply:
We definitely don't want to just always apply the same transform though

Reply:
ok, I was assuming for the case to call multiple times `KorniaParallelTransforms`